<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NVENCGUIClass</class>
 <widget class="QMainWindow" name="NVENCGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NVENCGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>341</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>224</width>
      <height>392</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelInput">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input file</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEditInput"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonInput">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelOutputDir">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output directory</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEditOutput"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonOutput">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelOutputFileName">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output file name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOutputFile">
       <property name="text">
        <string>.h264</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPreset">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPreset"/>
     </item>
     <item>
      <widget class="QLabel" name="labelBitrate">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bitrate</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="spinBoxBitrate">
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>300000</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelKbps">
         <property name="text">
          <string>kbps</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelResolution">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBoxWidth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1920</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelX">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxHeight">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxSameResolution">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Same as source</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelFrameRate">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Frame Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBoxFps">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>240</number>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>fps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxSameFps">
         <property name="text">
          <string>Same as source</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Encode!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>plainTextEdit</zorder>
   <zorder>progressBar</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>labelFrameRate</zorder>
   <zorder>spinBoxFps</zorder>
   <zorder>label</zorder>
   <zorder>checkBoxSameFps</zorder>
   <zorder>checkBoxSameFps</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="nvencgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>NVENCGUIClass</receiver>
   <slot>EncodeButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonInput</sender>
   <signal>clicked()</signal>
   <receiver>NVENCGUIClass</receiver>
   <slot>SetInputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOutput</sender>
   <signal>clicked()</signal>
   <receiver>NVENCGUIClass</receiver>
   <slot>SetOutputFileDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>EncodeButtonClick()</slot>
  <slot>SetInputFile()</slot>
  <slot>SetOutputFileDirectory()</slot>
 </slots>
</ui>
